lib/rucio/core/replica.py

```diff
@@ -1659,6 +1663,8 @@ def update_replicas_states(replicas, nowait=False, session=None):
                                             broken_rule_id=replica.get('broken_rule_id', None),
                                             broken_message=replica.get('broken_message', None),
                                             nowait=nowait, session=session)
+        elif replica['state'] == ReplicaState.TEMPORARY_UNAVAILABLE:
+            query = query.filter_by(state=ReplicaState.AVAILABLE)
```

Comment:
I don't get this line

Reply:
This is to make sure that the state transition can only go AVAILABLE -> TU